jenkins/scripts/utils.sh

```diff
@@ -45,3 +45,51 @@ wait_for_ssh() {
 
   echo "SSH connection to host[${SERVER}] is up."
 }
+
+# Description:
+# Check that the VM is in working condition, e.g. that the file-system is
+# resized properly.
+#
+# Usage:
+#   vm_healthy <ssh_user> <ssh_key_path> <server>
+vm_healthy() {
+  local USER KEY SERVER
+
+  USER="${1:?}"
+  KEY="${2:?}"
+  SERVER="${3:?}"
+
+  # CentOS has a cloud-init error all the time so we cannot rely on cloud-init
+  # to check if it is healthy or not.
+  # TODO: Fix cloud-init error on centos so we can remove this special check
+  if [ "${IMAGE_OS}" == "centos" ]; then
```

Comment:
well since it is temporary, perhaps no need because I expect we will fully drop that func anyways once we have the actual fix. So, I'm fine with both ways.
/approve